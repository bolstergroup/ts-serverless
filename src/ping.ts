import { successResponse, runWarm } from './utils'

export default runWarm((request, context, callback) => {
  callback(null, successResponse('pong'))
})
