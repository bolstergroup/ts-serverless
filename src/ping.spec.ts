import ping from './ping'

describe('ping', () => {
  it('Given three arguments, executes', () => {
    const cb = jest.fn()
    ping({}, {}, cb)
    expect(cb).toBeCalled()
    expect(cb).toMatchSnapshot()
  })
})
