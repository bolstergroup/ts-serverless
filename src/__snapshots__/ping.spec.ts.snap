// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ping executes as expected 1`] = `
[MockFunction] {
  "calls": Array [
    Array [
      null,
      Object {
        "body": "\\"pong\\"",
        "statusCode": 200,
      },
    ],
  ],
}
`;
